tests/unit_test/channels/whatsapp_test.py

```
@@ -0,0 +1,270 @@
+        os.environ["system_file"] = "./tests/testing_data/system.yaml"
+        Utility.load_environment()
+        Utility.load_system_metadata()
+        db_url = Utility.environment['database']["url"]
+        pytest.db_url = db_url
+        connect(**Utility.mongoengine_connection(Utility.environment['database']["url"]))
+
+    @pytest.mark.asyncio
+    async def test_valid_order_message_request(self):
+        from kairon.chat.handlers.channels.whatsapp import Whatsapp, WhatsappBot
+        with patch.object(WhatsappBot, "mark_as_read"):
+            with patch.object(Whatsapp, "process_message") as mock_message:
```

Ruff

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)